### PR TITLE
Improved add to home screen

### PR DESCRIPTION
**Pull request by PaulKinlan**
@owencm @petele PTAL.

### CODE REVIEW

File: src/content/en/updates/2017/02/improved-add-to-home-screen.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+book_path: /web/updates/_book.yaml
+description: Progressive Web Apps are becoming more integrated with your device. Learn how.
+
+{# wf_updated_on: 2017-02-02 #}
+{# wf_published_on: 2017-02-02 #}
+{# wf_tags: pwa,chrome57 #}
+{# wf_featured_image: /web/updates/images/generic/devices.png #}
+{# wf_featured_snippet: Progressive Web Apps are becoming more integrated with your device. Learn how. #}
+
+# The new and improved Add to Home screen {: .page-title }
```
Comment (petele) [inline src/content/en/updates/2017/02/improved-add-to-home-screen.md]: Titles should be Capital Cased.

File: src/content/en/updates/2017/02/improved-add-to-home-screen.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+
+{% include "web/_shared/contributors/paulkinlan.html" %}
+
+Chrome first introduced the ["Add to Home
+screen"](https://developers.google.com/web/fundamentals/engage-and-retain/app-install-banners/)[
```
Comment (petele) [inline src/content/en/updates/2017/02/improved-add-to-home-screen.md]: Do not use FQDNs for g.d.c, use only abs paths

File: src/content/en/updates/2017/02/improved-add-to-home-screen.md [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+**You can test your site by following these steps:**
+
+1. Install the latest [Chrome
+   Dev](https://play.google.com/store/apps/details?id=com.chrome.dev) or [Chrome
+   Beta](https://play.google.com/store/apps/details?id=com.chrome.beta) from the
+   Play Store if you don't have it already.
+1. Enable improved add to Home screen. These steps are only needed during the 
+   developer preview.
+    1. Open chrome://flags in Chrome dev and enable the flag 
+       \#enable-improved-a2hs ("Find in page" in the triple-dot menu is helpful
```
Comment (petele) [inline src/content/en/updates/2017/02/improved-add-to-home-screen.md]: intentional backslash?

File: src/content/en/updates/2017/02/improved-add-to-home-screen.md [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+This can cause some issues because if the user clears the Chrome profile, then 
+your data in your app will also be cleared. To ensure that your user data is 
+held more permanently, please use the [Persistent 
+Storage](https://developers.google.com/web/updates/2016/06/persistent-storage)[ 
+API](https://developers.google.com/web/updates/2016/06/persistent-storage).
+
+Please let us know if you have any feedback or questions. If you encounter a 
+bug, you can file it on the Chromium bug tracker 
+[here](https://bugs.chromium.org/p/chromium/issues/entry?components=Mobile%3EWebAPKs&labels=OS-Android,Type-Bug,Pri-2,Source-devpreview-feedback&cc=sbirch@google.com&comment=Android%20version%3A%0AChrome%20version%3A%20%0A%0ASite%3A%20%0A%0ASteps%20to%20reproduce%3A%0A(1)%0A(2)%0A(3)%0A%0AExpected%20result%3A%0A%0A%0AActual%20result%3A%0A). 
+Please also take a look at **FAQs below**** **that aim to answer any additional
```
Comment (petele) [inline src/content/en/updates/2017/02/improved-add-to-home-screen.md]: Weird bolding/emphasis here.

File: src/content/en/updates/2017/02/improved-add-to-home-screen.md (hunk 3 above)
Comment (PaulKinlan) [inline src/content/en/updates/2017/02/improved-add-to-home-screen.md]: nope.

**Pull request merged**